release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+definition_ is a built-in API that enables the ability to plug in your own
+custom, managed object and application as if it were native to Kubernetes.
+Therefore, you can leverage Kubernetes cluster management, RBAC and
+authentication services,  PI services, CLI, security, and so on, without having
+to know Kubernetes internals or modifying Kubernetes itself in any way.
+
+Custom Resource Definitions (CRD) deprecates Third Party Resources in Kubernetes
+1.7.
+
+How it works:
+
+. Define a CRD class (your custom objects) and register the new resource type.
+This defines how it fits into the hierarchy and how it will be referenced from
+the CLI and API.
+
+. Define a function to create a custom client, which is aware of the new resource
+schema.
+
+. Once completed, it can be accessed from the CLI. However, in order to build
+controllers or custom functionality, you need API access to the objects, and so
+you need to build a set of CRUD functions (library) to access the objects and the event-driven listener for controllers.
+
+. Create a client that:
++
+* Connects to the Kubernetes cluster.
+* Creates the new CRD (if it does not exist).
+* Creates a new custom client.
+* Creates a new test object using the client library.
+* Creates a controller that listens to events associated with new resources.
+
+See
+xref:../admin_guide/custom_resource_definitions.adoc#admin-guide-custom-resources[Extending
+the Kubernetes API with Custom Resources] for more information.
+
+[[ocp-37-api-aggregation]]
+==== API Aggregation
+
+There is now Kubernetes documentation on how API aggregation works in
+{product-title} 3.7 and how other users can add third-party APIs:
+
+* link:https://github.com/kubernetes/website/blob/master/docs/tasks/access-kubernetes-api/setup-extension-api-server.md[Set up an extension `api-server` to work with the aggregation layer]
+* link:https://github.com/kubernetes/website/blob/master/docs/concepts/api-extension/apiserver-aggregation.md[Kubernetes aggregation layer]
+
+[[ocp-37-master-prometheuh-endpoint-coverage]]
+==== Master Prometheus Endpoint Coverage
+
+Prometheus endpoint logic was added to upstream components so that monitoring
+and health indicators can be added around deployment configurations.
+
+[[ocp-37-end-to-end-provider-integration]]
+=== End-to-end Provider Integration
+
+[[ocp-37-deploying-cns-for-opsenshift-36]]
+==== Deploying Container-Native Storage for OpenShift Container Platform 3.6
+
+The
+link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.3/html/container-native_storage_for_openshift_container_platform/[Deploying
+Container-Native Storage for OpenShift Container Platform 3.6] guide describes
+the prerequisites and provides step-by-step instructions to deploy
+container-native storage (Red Hat Gluster Storage 3.3) with {product-title}.
+
+[[ocp-37-openshift-for-ops-test-drive]]
+==== OpenShift for Ops Test Drive
```

Comment:
How is this different from https://www.openshift.com/container-platform/trial.html or related to this? Should we point to this link instead?